### PR TITLE
clean components

### PR DESCRIPTION
**Pull request by gkueny**
This PR is a first pass to :
* clean components imports
* get ride of  styled-bootstrap-grid

Next steps are (in another PR) : 
* Rewrite components to be more atomic 
* Use `sass` instead of `styled-components` (see https://github.com/Gatsbylius/gatsbylius-starter/issues/1#issuecomment-616082234)

### CODE REVIEW

**Comment by gkueny** [comment]
I merge this PR. Feel free to review & comment, I will fix in other PR if necessary

File: src/services/checkout/submitCompleteCheckout.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+import axios from "axios";
 const SYLIUS_URL = process.env.GATSBY_SYLIUS_URL;
 
 export const submitCompleteCheckout = async storeState => {
   if (storeState.cartKey) {
     return await axios
       .put(`${SYLIUS_URL}/shop-api/checkout/${storeState.cartKey}/complete`, {
         email: "maher.isnard@gmail.com",
-        notes: "I'm await between 6 and 9pm",
+        notes: "I'm await between 6 and 9pm"
```
Comment (imhaage) [inline src/services/checkout/submitCompleteCheckout.js:10]: we'll have to make some changes here

File: src/templates/category.js [hunk 2]
```diff
@@ -1,18 +1,47 @@
...
+    width: 200px;
+  }
+`;
+
+const ProductsContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  flex: 1rem;
+  text-align: center;
+`;
```
Comment (imhaage) [inline src/templates/category.js:44]: This code should be placed in a styled.js file to maintain consistency

File: src/components/Checkout/Customer/CustomerPayment/CustomerPayment.js [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
+  useCheckoutStateContext
+} from "context/CheckoutContext";
+import { submitCustomerPayment } from "services/checkout/submitCustomerPayment";
+import { submitCompleteCheckout } from "services/checkout/submitCompleteCheckout";
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+`;
```
[the remark this answers is not in the document]
Comment (gkueny) [reply src/components/Checkout/Customer/CustomerPayment/CustomerPayment.js:28]: I agree. 
By lazyness I keep this styled component here. 

In our future refactoring we will have to be more strict and keep styled component in styled.js file

File: src/components/Checkout/Customer/CustomerShipping/CustomerShipping.js [hunk 4]
```diff
@@ -1,26 +1,31 @@
...
+import { priceParser } from "helpers/cartHelper";
 import { RadioGroup, ReversedRadioButton } from "react-radio-buttons";
-import { submitCustomerShipping } from "../../../../services/checkout/submitCustomerShipping";
+import { submitCustomerShipping } from "services/checkout/submitCustomerShipping";
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+`;
```
[the remark this answers is not in the document]
Comment (gkueny) [reply src/components/Checkout/Customer/CustomerShipping/CustomerShipping.js:28]: I agree with 
> reusable components.

But I'm not sure about `shared`. In my opinion, all components **must** be reusable. 

I think, we can use `atomic` design for this repository like `Storefront-ui`. But it's needs to be debated outside this PR.

**Pull request merged**